new/PRS14031John.xml

```diff
@@ -0,0 +1,66 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14031John" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>John of Parallos</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-05-30">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">John of Parallos</persName>
+                    <persName xml:lang="gez" xml:id="n2"><roleName type="title">አባ፡ </roleName>ዮሐንስ፡ </persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2"><roleName type="title">Abbā</roleName> Yoḥannəs</persName>
+                    <persName xml:lang="gez" xml:id="n3"><roleName type="title">አባ፡ </roleName>ዮሐንስ፡ ዘሀገረ፡ ቡርልስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n3"><roleName type="title">Abbā</roleName> Yoḥannəs zahagara Burləs</persName>
+                    <faith type="Coptic"/>
+                    <occupation type="ecclesiastic">Bishop of Nikiu</occupation>
+                    <nationality type="Egypt"/>
+                    <birth>ca. 540</birth>
+                    <death>610 to 620</death>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="saws:isAttributedToAuthor" active="PRS14031John" passive="LIT2375Synaxa"/>                    
```

Comment:
align to paths
`<relation name="skos:exactMatch" active="****" passive="http://........"/>`

Reply:
You mean, I should align John to Paralos (https://atlas.paths-erc.eu/places/132)?

Reply:
yes. and do not forget, the relation `saws:isAttributedToAuthor` is for texts authored by someone

Reply:
John collected the stories and miracles and justified their inclusion to the synaxary. I think it is enough to grant him authorship. I assume, that most Coptic or Ethiopic authors had a similar method. What do you think?

Reply:
You can ONLY use the relation FROM work TO person
So if you have a Work record for a Miracle then THERE you add the authorship relation (this work is attributed to this author)
Please read the Guidelines on relations

Reply:
You mean, I must delete the relation pointing to the synaxary.

Reply:
OK but you must understand the semantic difference between `isAttributedToAuthor `and `isAttributedAuthorOf `

Reply:
I understand it as following: If it is stated in a manuscript or in a work, that some person has written the following text, than this is a case for "isAttributedToAuthor" - no matter, whether this attribution is realistically true. If there is a person, who is acknowledged, what means as far as we can know he or she is accepted to be the author of this text, that the relation can be determined with "isAttributedAuthorOf". Do you agree to this interpretation or is there another point to consider?

Reply:
No you get it wrong

a WORK is Attributed to an AUTHOR
an AUTHOR is Attributed Author Of a WORK

please re-read the guidelines that specify the direction of pointing active-passive

Reply:
in a PRS record you may, if you want, have `isAttributedAuthorOf`
in a LIT record you may, if needed, have `isAttributedToAuthor`

Reply:
the meaning of the word "attributed" is the same, but it is essential that you read the sentence

a text is attributed to a person
a person is the attributed author of a text

not a person is attributed to a text as you have now

Reply:
Yes, the direction is important. I changed it in the record and I will push it in a minute. Then you will see, if it is correct.

Reply:
@CarstenHoffmannMarburg now that the branch has been merged do not forget to delete the balsa safo doublet (by creating a new branch from master and a new PR)